File: symfony/bike/src/ProduitBundle/Entity/produit.php
<?php

namespace ProduitBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
/**
 * produit
 *
 * @ORM\Table(name="produit", indexes={@ORM\Index(name="FK_Lib_S", columns={"Lib_S"}), @ORM\Index(name="FK_cat", columns={"Lib_C"})})
 * @ORM\Entity(repositoryClass="ProduitBundle\Repository\produitRepository")
 */
class produit
{
    /**
     * @var integer
     *
     * @ORM\Column(name="ID_P", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idP;

    /**
     * @var string
     * @Assert\NotBlank
     * @Assert\Length(min=3)
     * @ORM\Column(name="Libelle", type="string", length=255, nullable=false)
     */
    private $libelle;

    /**
     *
     * @ORM\Column(name="Image", type="string", length=255, nullable=false)
     */
    private $image;

    /**
     * @var string
     *  @Assert\NotBlank
     * @Assert\Length(min=5)
     * @ORM\Column(name="Description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var float
     *
     * @Assert\Range(
     *      min = 1,
     *      max = 999,
     *      minMessage = "Nombre petit",
     *      maxMessage = "Nombre grand"
     * )
     * @ORM\Column(name="Prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="Lib_C", type="string", length=255, nullable=false)
     */
    private $libC;

    /**
     * @var string
     *
     * @ORM\Column(name="Lib_S", type="string", length=255, nullable=false)
     */
    private $libS;

    /**
     * @return int
     */
    public function getIdP()
    {
        return $this->idP;
    }

    /**
     * @param int $idP
     */
    public function setIdP(int $idP)
    {
        $this->idP = $idP;
    }

    /**
     * @return string
     */
    public function getLibelle()
    {
        return $this->libelle;
    }

    /**
     * @param string $libelle
     */
    public function setLibelle(string $libelle)
    {
        $this->libelle = $libelle;
    }

    /**
     * @return mixed
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * @param mixed $image
     */
    public function setImage( $image)
    {
        $this->image = $image;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription(string $description)
    {
        $this->description = $description;
    }

    /**
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * @param float $prix
     */
    public function setPrix(float $prix)
    {
        $this->prix = $prix;
    }

    /**
     * @return string
     */
    public function getLibC()
    {
        return $this->libC;
    }

    /**
     * @param string $libC
     */
    public function setLibC(string $libC)
    {
        $this->libC = $libC;
    }

    /**
     * @return string
     */
    public function getLibS()
    {
        return $this->libS;
    }

    /**
     * @param string $libS
     */
    public function setLibS(string $libS)
    {
        $this->libS = $libS;
    }




}

File: bikeMobile/src/models/Produit.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package models;



/**
 *
 * @author Rajia
 */
public class Produit {
    protected int id_p;
    protected String libelle;
    protected String Lib_C;
    protected String Lib_S;
   protected String image;
   protected String description;
   protected Float prix;

    public void setId_p(int id_p) {
        this.id_p = id_p;
    }

    public int getId_p() {
        return id_p;
    }

    public Produit(int id_p, String libelle, String Lib_C, String Lib_S, String image, String description, Float prix) {
        this.id_p = id_p;
        this.libelle = libelle;
        this.Lib_C = Lib_C;
        this.Lib_S = Lib_S;
        this.image = image;
        this.description = description;
        this.prix = prix;
    }

    public Produit(String libelle, String Lib_C, String Lib_S, String image, String description, Float prix) {
        this.libelle = libelle;
        this.Lib_C = Lib_C;
        this.Lib_S = Lib_S;
        this.image = image;
        this.description = description;
        this.prix = prix;
    }

    public String getLib_S() {
        return Lib_S;
    }

    public void setLib_S(String Lib_S) {
        this.Lib_S = Lib_S;
    }



    public String getLibelle() {
        return libelle;
    }

    public String getLib_C() {
        return Lib_C;
    }

    public String getImage() {
        return image;
    }

    public String getDescription() {
        return description;
    }

    public Float getPrix() {
        return prix;
    }

    public void setLibelle(String libelle) {
        this.libelle = libelle;
    }

    public void setLib_C(String Lib_C) {
        this.Lib_C = Lib_C;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setPrix(Float prix) {
        this.prix = prix;
    }


   
    @Override
    public String toString() {
        return "Produit{" + "libelle=" + libelle + ", Lib_C=" + Lib_C + ", Lib_S=" + Lib_S + ", image=" + image + ", description=" + description + ", prix=" + prix + '}';
    }

 
    
}
